README.md

```diff
@@ -1,5 +1,7 @@
 # .NET Core Libraries (CoreFX)
 
+[![Build Status](https://dev.azure.com/dnceng/internal/_apis/build/status/dotnet/corefx/corefx-official?branchName=master&label=Build)](https://dev.azure.com/dnceng/internal/_build/latest?definitionId=283&branchName=master)
```

Comment:
I would make the label to say `Azure Pipelines` just like in the VSCode repo.

Reply:
Ok, sounds good. I was following Sams suggestion :smile:

Reply:
The short name makes sense with many badges (ie it is fine to use on the page that will have badge for everything). If you have just one badge, I think it looks better the way VSCode has it.

Reply:
Fixed.